azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -873,6 +874,29 @@ public void testFetchActiveFlowByExecId() throws Exception {
     Assert.assertEquals(flow1.getVersion(), execFlow1.getVersion());
   }
 
+  @Test
+  public void testFetchRecentlyFinishedFlows() throws Exception {
+    if (!isTestSetup()) {
```

Comment:
My understanding is that isTestSetup will return false in Travis effectively disabling this test. Am I right?